Assets/MRTK/Extensions/SceneTransitionService/AssemblyInfo.cs

```diff
@@ -8,3 +8,5 @@
 
 [assembly: AssemblyProduct("Microsoft® Mixed Reality Toolkit Extensions")]
 [assembly: AssemblyCopyright("Copyright © Microsoft Corporation")]
+
+[assembly: UnityEngine.Scripting.AlwaysLinkAssembly]
```

Comment:
Sorry if I missed a conversation on this entirely, but what do these do now? I thought we were reverting to the link.xml?

Reply:
This ensures the assembly is always processed by the linker. With the forthcoming PR to add an mrtk specific linker to generated folder, i can likely remove these.

Reply:
removed